src/tests/functional/plugin/conformance/test_runner/conformance_infra/include/conformance.hpp

```diff
@@ -44,19 +46,49 @@ inline ov::AnyMap readPluginConfig(const std::string &configFilePath) {
     return config;
 }
 
-inline std::vector<std::string> getModelPaths(const std::vector<std::string>& conformance_ir_paths) {
+inline std::vector<std::string> getModelPaths(const std::vector<std::string>& conformance_ir_paths,
+                                              const std::string opName = "") {
+    static std::list<std::string> dirList;
+    static bool listPrepared = false;
+    if (!listPrepared) {
+        // Looking for any applicable files in a folders
+        for (const auto& conformance_ir_path : conformance_ir_paths) {
+            std::vector<std::string> tmp_buf;
+            if (CommonTestUtils::directoryExists(conformance_ir_path)) {
```

Comment:
use `fileExist` before folder. It could affect the conformance run time

Reply:
Not sure I got your point, but change order isn't a problem.

Reply:
We need check lst file existing before to take folder. If it's file we should just read it instead of recursive search